client/index.html

```diff
@@ -468,6 +468,53 @@ <h1 class="title">Help</h1>
 							</div>
 						</div>
 
+						<h2>Autocompletion</h2>
+
+						<p>
+							Pressing <kbd>Tab</kbd> will autocomplete nicknames, channels, and <kbd>/</kbd>-commands.
+							Users with previous IRC experience will find this familiar; most clients implement tab-completion.
+						</p>
+						<p>
+							You can also start typing one of the following to get a menu of suggestions.
+							This is enabled by default, but can be turned off in settings if unwanted.
+						</p>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>@</code>
+							</div>
+							<div class="description">
+								<p>Nickname</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>#</code>
+							</div>
+							<div class="description">
+								<p>Channel</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/</code>
+							</div>
+							<div class="description">
+								<p>Commands (see list of commands below)</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>:</code>
+							</div>
+							<div class="description">
+								<p>Emoji</p>
+							</div>
+						</div>
```

Comment:
Add foreground/background color completion?

Reply:
Yes, definitely in the next revision. :smile: